docker-compose.yml

```diff
@@ -2,13 +2,13 @@
 version: '3'
 services:
     elasticsearch:
-        container_name: elasticsearch
+        container_name: microcosm_elasticsearch
```

Comment:
What's the rationale for the name change?

Reply:
It doesn't clash with other services that have their own compose files